### PR TITLE
Add TodoMVP example

### PR DESCRIPTION
**Pull request by ohanhi**
Your repo was missing the canonical TodoMVP example, so I added one written in JS6!

### CODE REVIEW

File: examples/todomvp/todomvp.js6 [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import WTF from '../lib/index̦';
+
+const funker = new WTF.Funk(undefined);
```
Comment (staltz) [inline examples/todomvp/todomvp.js6]: Please add some trailing spaces.

File: examples/todomvp/todomvp.js6 [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+funker.configuration = { transformer: new Optimus('Prime'), options: 'yes' };
+funker.setupFunk();
+
+// JS6 classes make everything fitter, better and happier!
+class Optimus {
+  constructor (feat) {
+    feat !== 'Prime' && throw new TypeError('Must be Prime.');
```
Comment (staltz) [inline examples/todomvp/todomvp.js6]: Please use `!=` instead of `!==`.

File: examples/todomvp/todomvp.js6 (hunk 1 above)
Comment (ohanhi) [inline examples/todomvp/todomvp.js6]: See https://github.com/ohanhi/wtf/commit/da5a9cf45c4367340d7595d3bd613e5284ea182c

File: examples/todomvp/todomvp.js6 (hunk 2 above)
Comment (ohanhi) [inline examples/todomvp/todomvp.js6]: C https://github.com/ohanhi/wtf/commit/37c919233ce1ed692f3510cc916ccdc2af8143a7

**Comment by staltz** [comment]
Thanks

**Pull request merged**